store/organizations.js

```diff
@@ -23,6 +28,29 @@ const $mutations = {
 
   [UPDATE_CACHE](state, organizations = []) {
     state.organizationCache = _.unionBy(organizations, state.organizationCache, (o) => o.organizationId);
+    _.forEach(organizations, (organization) => {
```

Comment:
you can also use native forEach of javascript array
```suggestion
    organizations.forEach((organization) => {
```

Reply:
Used native foreach